packages/light-client/src/update.ts

```diff
@@ -0,0 +1,84 @@
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {LIGHT_CLIENT_UPDATE_TIMEOUT} from "@chainsafe/lodestar-params";
+import {altair, Slot} from "@chainsafe/lodestar-types";
+import {intDiv} from "@chainsafe/lodestar-utils";
+import {List} from "@chainsafe/ssz";
+import {computeEpochAtSlot} from "@chainsafe/lodestar-beacon-state-transition";
+import {validateAltairUpdate} from "./validation";
+import {sumBits} from "./utils";
+
+/**
+ * A light client maintains its state in a store object of type LightClientStore and receives update objects of type LightClientUpdate.
+ * Every update triggers process_light_client_update(store, update, current_slot) where current_slot is the current slot based on some local clock.
+ * Spec v1.0.1
+ */
+export function processAltairUpdate(
```

Comment:
Can you rename `*AltairUpdate` and `*AltairStore` to `*LightClientUpdate` and `*LightClientStore`?

Reply:
Yeah much nicer